<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgConf</class>
 <widget class="QDialog" name="DlgConf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabConf">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>741</width>
     <height>391</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>741</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <property name="documentMode">
    <bool>false</bool>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="tabBarAutoHide">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="tabGroup">
    <attribute name="title">
     <string>Group</string>
    </attribute>
    <widget class="QTableWidget" name="tblGroup">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>631</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblSearch">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Search:</string>
     </property>
     <property name="buddy">
      <cstring>leSearch</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="leSearch">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>-3</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pbAddGrp">
     <property name="geometry">
      <rect>
       <x>553</x>
       <y>0</y>
       <width>41</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbDel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>320</y>
       <width>84</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbCancel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>320</y>
       <width>84</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblGroupName">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>0</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter Group Name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leGroupName">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>0</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabSchedule">
    <attribute name="title">
     <string>View</string>
    </attribute>
    <widget class="QTableWidget" name="tblSchedule">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>731</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pbCreate">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>300</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Create</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbDel_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>300</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbCancel_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>300</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbStart">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>300</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabCreate">
    <attribute name="title">
     <string>Create</string>
    </attribute>
    <widget class="QLabel" name="lblCrConfName">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblCrConfGrp">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Participant Group:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblCrConfDate">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Schedule Time:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbCrConfSave">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>240</y>
       <width>84</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbCrConfCancel">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>240</y>
       <width>84</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leCrConfName">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>311</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbCrConfGrp">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>311</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dateTimeEdit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>301</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>120</y>
       <width>145</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="rbCrConfIn">
        <property name="text">
         <string>Dial-in </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbCrConfOut">
        <property name="text">
         <string>Dial-out</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>150</y>
       <width>124</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="rbCrConfAudio">
        <property name="text">
         <string>Audio</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbCrConfVideo">
        <property name="text">
         <string>Video</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>180</y>
       <width>259</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QRadioButton" name="rbCrConfWithMod">
        <property name="text">
         <string>With Moderator</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbCrConfWithoutMod">
        <property name="text">
         <string>Without Moderator</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>210</y>
       <width>172</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QRadioButton" name="rbCrConfRec">
        <property name="text">
         <string>Record</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbCrConfNoRec">
        <property name="text">
         <string>Don't Record</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tabStart">
    <attribute name="title">
     <string>Start</string>
    </attribute>
    <widget class="QTableWidget" name="tblStartConf">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>561</width>
       <height>311</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>598</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="pbStartStart">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbStartReconnect">
        <property name="text">
         <string>Reconnect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbStartMute">
        <property name="text">
         <string>Mute</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbStartUnmute">
        <property name="text">
         <string>Unmute</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbStartKick">
        <property name="text">
         <string>Kickout</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbStartBreakout">
        <property name="text">
         <string>Breakout</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbStartRejoin">
        <property name="text">
         <string>Rejoin</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QOpenGLWidget" name="oglConf">
     <property name="geometry">
      <rect>
       <x>559</x>
       <y>10</y>
       <width>171</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
